coaster/sqlalchemy/statemanager.py

```diff
@@ -230,6 +244,14 @@ class StateTransitionError(BadRequest, TypeError):
     pass
 
 
+class AbortTransitionError(Exception):
+    """
+    Raised by the transition function to abort the transition,
+    in which case the state does not change
```

Comment:
Transitions are methods, not functions. This docstring should be: Transitions may raise AbortTransition to return without changing state. The parameters to the exception are returned as the transition’s result.